test/parallel_api/numeric/numeric.ops/transform_reduce.pass.cpp

```diff
@@ -71,21 +72,35 @@ class MyClass
 
 template <typename T>
 void
-CheckResults(const T& expected, const T& in)
+CheckResults(const T& expected, const T& in, const ::std::string& msg)
```

Comment:
Does it make sense to use const std::string instead of `const char*` here? I believe using `const char*` would be much simpler.